### ISSUE TITLE
split_equally option when creating an expense

### ISSUE THREAD

**Issue by rougeth** [namaggarwal/splitwise#29]
Hello! Is the `split_equally` option when creating an expense available in the library? According to the API documentation:

> If you set a group_id and split_equally to true the expense will be split equally with all members of the group

https://dev.splitwise.com/#create_expense

**Comment by namaggarwal** [comment]
Hello,

Apologies for late reply. Split equally option is not there. I am working on a new version which will be out by this week which will have all the options.

You will be able to use it like this
```
expense = Expense()
expense.setSplitEqually()
expense, errors = sObj.createExpense(expense)
```

**Comment by rougeth** [comment]
Great! Thanks @namaggarwal!

**Comment by namaggarwal** [comment]
This is available since version 2.0.0

Closing this issue
